ppb/__init__.py

```diff
@@ -23,13 +34,10 @@ def run(setup: Callable[[BaseScene], None]=None, *, log_level=logging.WARNING,
     """
     logging.basicConfig(level=log_level)
 
-    kwargs = {
-        "resolution": (800, 600),
-        "scene_kwargs": {
-            "set_up": setup,
-        },
-        "window_title": title,
-    }
-
-    with GameEngine(starting_scene, **kwargs) as eng:
+    with GameEngine(starting_scene, **_make_kwargs(setup, title)) as eng:
```

Comment:
Why not just call `make_engine()`?

Reply:
Mostly because it didn't seem like it would make the code any shorter or clearer...